src/plugins/wizard/public/application/components/workspace.tsx

```diff
@@ -66,13 +87,13 @@ export const Workspace: FC = ({ children }) => {
       </EuiFlexGroup>
       <EuiPanel className="wizCanvas">
         {expression ? (
-          <ReactExpressionRenderer expression={expression} />
+          <ReactExpressionRenderer expression={expression} searchContext={searchContext} />
```

Comment:
`ReactExpressionRenderer` has a prop `reload$`:
```
An observable which can be used to re-run the expression without destroying the component
```
Does it make sense to use that instead of an arbitrary prop `searchContext`?

Reply:
Yeah, for now either the `searchContext` or `expression` will need to change causing a reload anyways. But thanks for calling that out. This will be useful in instances where neither change but you still want to trigger a reload.